x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -1096,98 +1065,82 @@ describe.skip('AllCasesListGeneric', () => {
       });
 
       it('should disable row actions when bulk selecting all cases', async () => {
-        const res = appMockRenderer.render(<AllCasesList />);
+        appMockRenderer.render(<AllCasesList />);
 
         act(() => {
-          userEvent.click(res.getByTestId('checkboxSelectAll'));
+          userEvent.click(screen.getByTestId('checkboxSelectAll'));
         });
 
         await waitFor(() => {
           for (const theCase of defaultGetCases.data.cases) {
-            expect(res.getByTestId(`case-action-popover-button-${theCase.id}`)).toBeDisabled();
+            expect(screen.getByTestId(`case-action-popover-button-${theCase.id}`)).toBeDisabled();
           }
         });
       });
 
       it('should disable row actions when selecting a case', async () => {
-        const res = appMockRenderer.render(<AllCasesList />);
+        appMockRenderer.render(<AllCasesList />);
         const caseToSelect = defaultGetCases.data.cases[0];
 
         act(() => {
-          userEvent.click(res.getByTestId(`checkboxSelectRow-${caseToSelect.id}`));
+          userEvent.click(screen.getByTestId(`checkboxSelectRow-${caseToSelect.id}`));
         });
 
         await waitFor(() => {
           for (const theCase of defaultGetCases.data.cases) {
-            expect(res.getByTestId(`case-action-popover-button-${theCase.id}`)).toBeDisabled();
+            expect(screen.getByTestId(`case-action-popover-button-${theCase.id}`)).toBeDisabled();
           }
         });
       });
     });
-  });
-});
 
-// Flaky: https://github.com/elastic/kibana/issues/148490
-describe.skip('Assignees', () => {
```

Comment:
This block was outside of main describe block which caused timeout to get mock data of `useGetCases`